app_data/admin.py

```diff
@@ -81,15 +81,16 @@ def _get_form_factory_opts(self, request, obj=None, **kwargs):
 
 
 class AppDataModelAdmin(AppDataAdminMixin, ModelAdmin):
-    def get_form(self, request, obj=None, **kwargs):
+    def get_form(self, request, obj=None, change=False, **kwargs):
```

Comment:
isn't the generic `**kwargs` enough here, given that we pass this unchanged to the super